src/engine/runtime.js

```diff
@@ -2015,6 +2020,14 @@ class Runtime extends EventEmitter {
         }
     }
 
+    /**
+     * Reset the Run ID. Call this any time the project logically starts, stops, or changes identity.
+     */
+    resetRunId () {
+        const newRunId = uuid.v1();
```

Comment:
Instead of adding a dependency on `uuid`, why not just use `util/uid`?

Reply:
I actually used `util/uid` in an earlier version :)

I made the change based on a request from another member of the Scratch team. I think using a standard data format gives us a little more flexibility. Also, the `uuid` module was already in our dependency tree, just not at the top level.

Reply:
Did we pull it into the top level to make it explicit then? It might be worth doing that if not so we can lock versions of it properly and it's good code health thing to do anyways for explicit documentation and maintenance. @cwillisf

Reply:
Nvm, just saw the code change, ignore me!